manifests/plugins_dir.pp

```diff
@@ -14,7 +14,7 @@
       recurse => $recurse,
       purge   => $purge,
       force   => $force,
-      require => Package['sensu'],
+      require => Package[$::sensu::package::package_name],
     }
```

Comment:
sensu-enterprise replaces sensu-server and sensu-api. sensu-client (OSS),
among other things, still requires the sensu package.
sensu-enterprise-dashboard replaces uchiwa.

On Mon, Aug 10, 2015 at 3:29 PM Justin Lambert <notifications@github.com>
wrote:

> In manifests/plugins_dir.pp
> <https://github.com/sensu/sensu-puppet/pull/400#discussion_r36692707>:
>
> > @@ -14,7 +14,7 @@
> >        recurse => $recurse,
> >        purge   => $purge,
> >        force   => $force,
> > -      require => Package['sensu'],
> > +      require => Package[$::sensu::package::package_name],
> >      }
>
> Ahhh - this may also explain the above comment. What components does SE
> replace? Is it (essentially) just the dashboard addition to OSS?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sensu/sensu-puppet/pull/400/files#r36692707>.
>